server-protocol.md

```diff
@@ -92,6 +85,23 @@ following keys (and ignores all others):
   future version of the protocol requires a rate-limiting CAPTCHA ticket or
   other authorization record, the server can send `error` (explaining the
   requirement) if it does not see this ticket arrive before the `bind`.
+* `permission-required`: various ways the server supports for clients
+  to request permission to use the service currently.
+
+A client that supports asking for permission (see also "Permission and
```

Comment:
Wording: IMO, a client *supports* (<-- imperative) this feature. Clients that don't should be mentioned as "backwards compatibility". People reading the spec will do so from the perspective of a full implementation as goal.

Reply:
okay, I re-worded this paragraph entirely